qt/widgets/instrumentview/src/InstrumentWidgetRenderTab.cpp

```diff
@@ -187,7 +192,7 @@ QPushButton *InstrumentWidgetRenderTab::setupDisplaySettings() {
   m_maintainAspectRatio = new QAction("Maintain Aspect Ratio", this);
   m_maintainAspectRatio->setCheckable(true);
   m_maintainAspectRatio->setChecked(true);
-  connect(m_maintainAspectRatio, SIGNAL(toggled(bool)), m_instrWidget, SLOT(setMaintainAspectRatio(bool)));
+  connect(m_maintainAspectRatio, SIGNAL(toggled(bool)), this, SLOT(setMaintainAspectRatio(bool)));
```

Comment:
The reason I implemented essentially the same method on `this` class is because otherwise sip was not finding the bindings